contracts/tournament/Tournament.cairo

```diff
@@ -5,17 +5,27 @@
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.bool import (TRUE, FALSE)
-from starkware.cairo.common.math import (assert_lt, assert_nn)
+from starkware.cairo.common.math import (assert_lt, assert_nn, assert_not_zero)
+from starkware.cairo.common.math_cmp import is_le
 from starkware.cairo.common.uint256 import (Uint256, uint256_le)
 from starkware.starknet.common.syscalls import (get_contract_address, get_caller_address)
+from starkware.cairo.common.math import unsigned_div_rem
+from starkware.cairo.common.alloc import alloc
 
 # OpenZeppeling dependencies
 from openzeppelin.access.ownable import (Ownable_initializer, Ownable_only_owner)
 from openzeppelin.token.erc20.interfaces.IERC20 import IERC20
 from openzeppelin.token.erc721.interfaces.IERC721 import IERC721
 
-from contracts.interfaces.ispace import ISpace
-
+#from contracts.interfaces.ispace import ISpace
+# TODO: remove this
```

Comment:
Can you actually remove it and import the interface

Reply:
ahah good catch :D